tests/components/hassio/test_http.py

```diff
@@ -155,6 +155,8 @@ def test_need_auth(hass):
     """Test if the requested path needs authentication."""
     assert not _need_auth(hass, "addons/test/logo")
     assert _need_auth(hass, "snapshots/new/upload")
+    assert _need_auth(hass, "supervisor/logs")
 
     hass.data["onboarding"] = False
     assert not _need_auth(hass, "snapshots/new/upload")
+    assert not _need_auth(hass, "supervisor/logs")
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6243615)